web/libs/datamanager/src/components/Common/MediaPlayer/MediaPlayer.scss

```
@@ -0,0 +1,68 @@
+.player {
+  width: 100%;
+  height: 32px;
+  display: flex;
+  cursor: default;
+  min-width: 240px;
+  border-radius: 4px;
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 0 0 1px rgba(#000, 0.2) inset, 0 0 2px 2px rgba(#000, 0.05);
```

Flagging as opportunity for css var replacement, further review of usage needed.
```suggestion
  box-shadow: 0 0 0 1px var(--black_20) inset, 0 0 2px 2px rgba(var(--black), 0.05);
```